app/models/queen.rb

```diff
@@ -1,13 +1,13 @@
 # Queen behavior.
 class Queen < Piece
-  # Capture, collision, check, and checkmate logic are not
+  # Capture, check, and checkmate logic are not
   # implemented yet and thus ignored.
   def valid_move?(x, y)
     moved?(x, y) &&
       (
```

Comment:
Cool, I actually tried that earlier in its own array just to test it, and Ruby couldn't parse it.  But it works now.